models/containerized_model/templates/environment.yml

```diff
@@ -0,0 +1,17 @@
+# An example for a model in s3 bucket
+#MODEL_PATH=s3://bucket/model.pkl
+
+# Default location for local model
+MODEL_PATH=/model/model.pkl
+
+# S3 credentials
+BUCKET_ENDPOINT=<your-s3-bucket-endpoint>
```

Comment:
For BUCKET_ENDPOINT, I'd suggest commenting it as it only needs filling in if its something other than standard aws. Could you also give an actual example e.g. for aws?  I'm not actually sure what to put here